internal/ent/schema/origin.go

```diff
@@ -40,6 +44,11 @@ func (Origin) Fields() []ent.Field {
 			Annotations(
 				entgql.OrderField("name"),
 			),
+		field.Int("weight").
```

Comment:
do we want to put any range constraints on the weight values that can persist?

Reply:
Yeah, I think you about covered it.Different providers accept it in all kinds of shapes and sizes, so putting too many constraints on this out of the gate will likely cause problems.

Reply:
I can't seem to find any providers that use floats for weights, but in the worst case it should be pretty easy to convert from int to float database and code-wise